source/fuzz/fuzzer_pass.h

```diff
@@ -273,6 +273,16 @@ class FuzzerPass {
   uint32_t FindOrCreateZeroConstant(uint32_t scalar_or_composite_type_id,
                                     bool is_irrelevant);
 
+  // Adds a pair {id_use_descriptor, |replacement_id|} to the vector
```

Comment:
I prefer round brackets - ( ... ) - for pairs, as in mathematics { ... } denotes an unordered set.

Reply:
Changed